sentry-ruby/lib/sentry/session.rb

```diff
@@ -17,9 +16,11 @@ def initialize
       @aggregation_key = Time.utc(@started.year, @started.month, @started.day, @started.hour, @started.min)
     end
 
-    # TODO-neel add :crashed after adding handled mechanism
-    def update_from_exception(_exception = nil)
+    def update_from_event(event = nil)
```

Comment:
Why are we calling it `update_from_event` but in fact there may be no event? It looks like the only place where it could be `nil` is in tests, can we improve the setup there so `event` becomes a required argument?